src/flows/context/flow.current.tsx

```diff
@@ -158,7 +168,9 @@ export const FlowProvider: FC = ({children}) => {
             ...flowCopy,
             ...flowCopy.meta.byID[id],
```

Comment:
idk why i missed this before, but why are we overwriting the data within the meta element, onto the parent container? i think we're just supposed to be writing flowCopy to the ydoc

Reply:
Could be. This is ephemeral pasta